linode/instance/tmpl/template.go

```diff
@@ -732,3 +733,20 @@ func WithPG(t *testing.T, label, region, assignedGroup string, groups []string)
 			AssignedGroup:   assignedGroup,
 		})
 }
+
+func WithReservedIP(t *testing.T, label, pubKey, region, rootPass string, reservedIP string) string {
+	generatedConfig := acceptance.ExecuteTemplate(t,
+		"instance_with_reserved_ip", TemplateData{
+			Label:    label,
+			PubKey:   pubKey,
+			Image:    acceptance.TestImageLatest,
+			Region:   region,
+			RootPass: rootPass,
+			IPv4:     []string{reservedIP},
+		})
+
+	// Add this debug logging
+	t.Logf("Generated Terraform config for WithReservedIP:\n%s", generatedConfig)
```

Comment:
We usually don't do debug logging for integration tests. Is there a specific reason that it's necessary?

Reply:
Yeah I am going to go ahead and remove it